engines/python/setup/djl_python/deepspeed.py

```diff
@@ -94,6 +94,11 @@
     "text2text-generation": AutoModelForSeq2SeqLM
 }
 
+OUTPUT_FORMATTER_TO_CONTENT_TYPE = {
```

Comment:
Can we move this constant to rolling_batch.py?

Reply:
Done